models/_sector/dex/trades/base/_schema.yml

```diff
@@ -766,3 +766,22 @@ models:
           seed_file: ref('gridex_base_base_trades_seed')
           filter:
             version: 1
+
+  - name: baseswap_basex_base_base_trades
```

Comment:
By the way,
do you think we may put `basex` in the version or project, instead of `3`?